TACC/Frontera-Portal
# task/FP427 - metadata replacement port

## Overview: ##

Work in progress

## PR Status: ##


## Related Jira tickets: ##

* [FP-427](https://jira.tacc.utexas.edu/browse/FP-427)

## Summary of Changes: ##

- Ported App Tray backend from CEPv1
- Added Admin interface (accessible at cep.dev/core/admin)

## Testing Steps: ##
1. Import existing metadata using python manage.py import-apps
2. Check to make sure all of the categories and app entries appear at cep.dev/core/admin in the App Tray Entrys and App Tray Categorys management

## UI Photos:

## Notes: ##

This has no front end changes yet.
